python/lancedb/db.py

```diff
@@ -52,12 +52,24 @@ def create_table(
         ----------
         name: str
             The name of the table.
-        data: list, tuple, dict, pd.DataFrame; optional
-            The data to initialize the table. User must provide at least one of `data` or `schema`.
-        schema: pyarrow.Schema or LanceModel; optional
-            The schema of the table.
+        data: The data to initialize the table, *optional*
+            User must provide at least one of `data` or `schema`.
+            Acceptable types are:
```

Comment:
Can we make sure this list rendered correctly in the doc

Reply:
The first screenshot attached presented the rendered doc. Does it look OK?